engine/NetworkProcessorNet.go

```diff
@@ -389,6 +390,12 @@ func Q2(fnode *FactomNode, source string, msg interfaces.IMsg) {
 	fnode.State.InMsgQueue2().Enqueue(msg)
 }
 
+func DataQ(fnode *FactomNode, source string, msg interfaces.IMsg) {
+	q := fnode.State.DataMsgQueue()
+	fnode.State.LogMessage("DataQueue", fmt.Sprintf(source+", enqueue %v", len(q)), msg)
+	q <- msg
```

Comment:
Do we want to drop requests if this backs up?

I don't see a deadlock every happening, but a ton of requests could bog down a node. I'd rather my node drop a request than block.

Reply:
I don't think this is configured w/ a max size